test/helpers/db.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const knex = require('../../src/lib/knex').knex
+const knexConfig = require('../../src/lib/knex').config
+const createTables = require('../../src/lib/db').createTables
+const dropTables = require('../../src/lib/db').dropTables
+
+const tables = [
+  'N_CASES',
+  'N_NOTIFICATIONS'
+]
+
+// Only run migrations once during tests
+let init = false
+exports.init = function * () {
+  if (init) {
+    return
+  }
+
+  yield createTables(knex, knexConfig)
+  init = true
+  return
+}
+
+exports.drop = function * () {
+  yield dropTables(knex, knexConfig)
+}
+
+exports.clean = function * () {
+  for (let t of tables) {
+    if (knex.client.config.client === 'pg') {
+      /* eslint-disable no-useless-escape */
+      yield knex.raw('TRUNCATE TABLE \"' + t + '\" CASCADE;').then()
+      /* eslint-enable no-useless-escape */
+    } else {
+      yield knex(t).truncate().then()
```

Comment:
Would be good to move `truncateTables` to `src/lib/db` too to keep direct access to `knex` out of the `test` directory